### ISSUE TITLE
Commandline parameters to up/down helpers seem unstable

### ISSUE THREAD

**Issue by tbaumann** [jonathanio/update-systemd-resolved#54]
My OpenVPN version (2.4.6) calls the up/down helpers with different arguments than this script expects
```
/etc/openvpn/scripts/update-systemd-resolved tun0 1500 1555 10.235.97.2 255.255.255.128 init
```
However, the variables dev and script_type are passed via ENV. I see no need to rely on parameter parsing.

**Comment by tbaumann** [comment]
log
```
Thu Oct 25 14:15:45 2018 TUN/TAP device tun0 opened
Thu Oct 25 14:15:45 2018 TUN/TAP TX queue length set to 100
Thu Oct 25 14:15:45 2018 do_ifconfig, tt->did_ifconfig_ipv6_setup=0
Thu Oct 25 14:15:45 2018 /sbin/ip link set dev tun0 up mtu 1500
Thu Oct 25 14:15:45 2018 /sbin/ip addr add dev tun0 10.235.96.135/25 broadcast 10.235.96.255
Thu Oct 25 14:15:45 2018 /etc/openvpn/scripts/update-systemd-resolved tun0 1500 1555 10.235.96.135 255.255.255.128 init
<11>Oct 25 14:15:45 update-systemd-resolved[9417]: Invalid script type: 'tun0'. Usage: update-systemd-resolved up|down device_name.
Thu Oct 25 14:15:45 2018 WARNING: Failed running command (--up/--down): external program exited with error status: 1
Thu Oct 25 14:15:45 2018 Exiting due to fatal error
```

**Comment by jonathanio** [comment]
As per #55 and the commit 55c7144d91f10ca7ef20372f731692e0798dfd10, this can now be closed.
